plugins/GSdx/GSRenderer.cpp

```diff
@@ -200,11 +221,12 @@ bool GSRenderer::Merge(int field)
 	{
 		if(!en[i] || !tex[i]) continue;
 
-		GSVector4i r = fr[i];
+		static std::array<GSVector4i, 2> r = { GSVector4i(0), GSVector4i(0) };
+		r[i] = DelayDisplayRectangleChange(fr[i], r[i], i);
```

Comment:
Hum, I think it would be possible to keep this code mostly unchanged. In my opinion, you should be able to do `GSVector4i r =  DelayDisplayRectangleChange(fr[i], i);`.